packages/friends/yml/en/john-pemberton.yml

```diff
@@ -1,15 +1,35 @@
 name: John Pemberton
 slug: john-pemberton
 gender: male
-description: TODO
+description:
+  The ​​simplicity, meekness and diffidence​ of John Pemberton (1727-1795)​, united with
```

Comment:
Same sort of reservations about using "Diffidence" here. Please overrule if you think i'm off base.

Reply:
yeah... you're right. changed it.